File: JULIA/Cap04/c04ex03.jl
# c04ex03.jl

print("Entre o 1o. valor numerico: ")
a = parse(Int, readline())

print("Entre o 2o. valor numerico: ")
b = parse(Int, readline())

x = a + b

if (x >= 10)
  r = x + 5
else
  r = x - 7
end

println("Resultado = $r")

println()
print("Tecle <Enter> para encerrar... ")
readline()

File: CSHARP/Cap04/c04ex03.cs
// c04ex03.cs

using System;
using static System.Console;

class Program
{
    static void Main(string[] args)
    {
        int A, B, X, R;

        Write("Entre o 1o. valor numerico: ");
        A = int.Parse(ReadLine());

        Write("Entre o 2o. valor numerico: ");
        B = int.Parse(ReadLine());

        X = A + B;

        if (X >= 10)
            R = X + 5;
        else
            R = X - 7;

        WriteLine("Resultado = " + R);

        Write("\nAperte <Enter> para encerrar... ");
        ReadLine();
    }
}
